spec/models/course_user_spec.rb

```diff
@@ -147,6 +161,33 @@
       end
     end
 
+    describe '#level_progress_percentage' do
+      subject { student.level_progress_percentage }
+      before do
+        create :course_level, course: course, experience_points_threshold: 100
+        course.reload
+      end
+
+      it { is_expected.to be_a(Fixnum) }
+      it { is_expected.to be_between(0, 100) }
+
+      context 'when the course user has 0% of progress within the level' do
+        it { should eq(0) }
```

Comment:
...I mean 'should' should not be used... because of syntactic inconsistencies they changed it to is_expected.